site/content/faq/what-is-the-default-directory-that-zap-uses.md

```diff
@@ -7,35 +7,21 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+Note that you can override this using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+The 'home' or 'user' directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, logs, etc. It is also the initial default save location.
```

Comment:
Please add what is saved in the initial default save location. Eg. Scripts, contexts active scan policy files sessions. etc